lib/terrafying/components/letsencrypt.rb

```diff
@@ -259,7 +259,8 @@ def renew(name, bucket, options={})
                       Action: [
                         "s3:Put*",
                         "s3:Get*",
-                        "s3:DeleteObject"
+                        "s3:DeleteObject",
+                        "s3:ListBucket"
```

Comment:
list can only be done on the bucket not on the path

Reply:
thanks, updated